src/Features/CSharp/Portable/CodeFixes/Iterator/CSharpChangeToIEnumerableCodeFixProvider.cs

```diff
@@ -32,11 +33,13 @@ public override ImmutableArray<string> FixableDiagnosticIds
         {
             var model = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
             var methodSymbol = model.GetDeclaredSymbol(node, cancellationToken) as IMethodSymbol;
```

Comment:
Probably worth a comment that methodSymbol can be either a regular method or an accessor and to be careful about syntax.